1-exercises/H-array-methods-2/exercise2.js

```diff
@@ -7,7 +7,11 @@
   Tip: use the string method .split() and the array method .join()
 */
 
-function capitalise(str) {}
+function capitalise(str) {
```

Comment:
You could have used slice as well. But yours is correct as well

Reply:
Thanks Farzad. It's interesting looking through the code and seeing the different ways people get to the same solution!